citi-onboarding/colisio
# Feature(crud-carousel-2): Crud working with its bugs fixed

## Created the Crud for the Carousel section.

- Created another branch in order to solve some bugs happening in the previous branch. 
   - Create, Read, Updade and Delete working.

## How to test

1. Open the terminal and go to the folder of this project

2. In terminal:
   `git checkout develop`
   `git pull`
   `git checkout feature/crud-carousel-2`
   `git pull origin crud-carousel-2`
   `yarn install`
   `yarn start`

1. In your browser go to http://localhost:port

1. Check if the component looks just like the mockup in different screen sizes 

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description
